guides/hidden-pages.mdx

```diff
@@ -0,0 +1,31 @@
+---
+title: "How to add hidden pages"
+description: "Exclude pages from your docs.json"
```

Comment:
```suggestion
title: "Hidden pages"
description: "Exclude pages from your `docs.json`"
icon: "eye-closed"
```

Titling it "hidden pages" is more similar to how other guides are named and let's add an icon.